src/dotnet-install.ps1

```diff
@@ -297,7 +306,7 @@ function Load-Assembly([string] $Assembly) {
     }
 }
 
-function GetHTTPResponse([Uri] $Uri, [bool]$HeaderOnly, [bool]$DisableRedirect)
+function GetHTTPResponse([Uri] $Uri, [bool]$HeaderOnly, [bool]$DisableRedirect, [bool]$DisableFeedCredential)
```

Comment:
Does the new parameter default to `false` when not specified?

Reply:
yes, the same as for the other parameters. I have checked that, sure.